flopy/mbase.py

```diff
@@ -1634,61 +1656,58 @@ def to_shapefile(self, filename, package_names=None, **kwargs):
         """
         warnings.warn("to_shapefile() is deprecated. use .export()")
         self.export(filename, package_names=package_names)
-        return
 
 
 def run_model(
-    exe_name,
-    namefile,
-    model_ws="./",
-    scrub=False,
+    exe_name: Union[str, os.PathLike],
+    namefile: Optional[str],
```

Comment:
Currently this is a required parameter, because it does not have a default e.g. `= None`.

Reply:
The docstring indicated `namefile=None` could be used for models that don't require namefiles &mdash; the function seems to handle this case OK, just omits it from the args passed to the executable